com.biglybt.BiglyBT.yaml

```
@@ -0,0 +1,56 @@
+app-id: com.biglybt.BiglyBT
+runtime: org.gnome.Platform
+runtime-version: '41'
```

```suggestion
runtime-version: '42'
```

Do you really need the GNOME platform? Isn't freedesktop-sdk enough since this is Java.

---

The UI was not showing correctly on freedeskop, the app uses SWT

---

so it should work on freedekstop-sdk.